AUTHORS

```diff
@@ -118,6 +120,7 @@ Jasper Deng <jasper.deng@gmail.com>
 Jaysinh Shukla <jaysinhp@gmail.com>
 Jenna Mandel <jenna.b.mandel@gmail.com>
 Jerry Chen <jerry.c@berkeley.edu>
+Jim <35385366+jimbyo@users.noreply.github.com>
```

Comment:
Should we add a proper email here? (not github reply email)

Reply:
Jop, I have this ready in https://github.com/vojtechjelinek/oppia/commits/update-changelog-for-releasev2.9.2 for some reason GitHub is having some problems ATM so it is not propageted here.